src/cliActions.js

```diff
@@ -136,9 +136,13 @@ const streamDApp = ({ pw }, signingKeyFile, devMode) =>
         // read signing key
         const rawSigningKey = fs.readFileSync(signingKeyFile, 'utf8');
         const signingKey = JSON.parse(rawSigningKey);
-        watchAndStreamBundleData(devMode, signingKey, ({ content }) => {
-          console.log({ content, singingPrivateKey, signingKey }); // TODO: need to process result
-        });
+        watchAndStreamBundleData(
+          devMode,
+          { ...signingKey, singingPrivateKey },
+          ({ content }) => {
+            console.log({ content, singingPrivateKey, signingKey }); // TODO: need to process result
```

Comment:
shouldn't the content be bundled and signed?

Reply:
done